common.sh

```diff
@@ -160,7 +160,10 @@ export MIRROR_LOG_FILE=${REGISTRY_DIR}/${CLUSTER_NAME}-image_mirror-${OPENSHIFT_
 # Switch Container Images to upstream, Installer defaults these to the openshift version
 if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export IRONIC_LOCAL_IMAGE=${IRONIC_LOCAL_IMAGE:-"quay.io/metal3-io/ironic:master"}
-    export IRONIC_INSPECTOR_LOCAL_IMAGE=${IRONIC_INSPECTOR_LOCAL_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+# Starting from Openshift 4.9 the ironic-inspector container is not used anymore
+    if [[ "$OPENSHIFT_VERSION" < "4.9" ]]; then
```

Comment:
You cannot use string comparison like this.
```
$ [[ "4.10" < "4.9" ]] && echo ok
ok
```

Reply:
:facepalm: 
I swear I know this......